src/both.js

```diff
@@ -26,7 +31,9 @@ var _curry2 = require('./internal/_curry2');
  *      f(101); //=> false
  */
 module.exports = _curry2(function both(f, g) {
-  return function _both() {
-    return f.apply(this, arguments) && g.apply(this, arguments);
-  };
+  return f instanceof Function ?
```

Comment:
Let's define `_isFunction` as we defined [`_isNumber`][1], and use it here.


[1]: https://github.com/ramda/ramda/blob/v0.19.0/src/internal/_isNumber.js